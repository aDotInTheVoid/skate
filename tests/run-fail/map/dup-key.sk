fn main() {
    print %{a:2, a:3, a:4}.a;
}
