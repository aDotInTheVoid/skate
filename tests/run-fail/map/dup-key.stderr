error: Duplicate key `a` in map literal
  ┌─ $DD/tests/run-fail/map/dup-key.sk:2:13
  │
2 │     print %{a:2, a:3, a:4}.a;
  │             ^    ^ Defined here again
  │             │     
  │             First defined here

